scss/blocks/figma-component-props-table.scss

```diff
@@ -0,0 +1,120 @@
+:root {
+    --figma-component-props-table-background-color: rgba(244, 245, 247, 0.95);
+    --figma-component-props-table-badge-border-color: #ECEEF3;
+}
+
+table.props-table {
+    width: 100%;
```

Comment:
The table columns width should be evenly distributed. `table-layout: fixed;` should do the trick

Reply:
Are you sure? The `Values` column seems to be wider than the other ones:
<img width="648" alt="image" src="https://github.com/user-attachments/assets/5973964d-2f3c-4510-bda5-9a6f3d42e9ba">

Reply:
[da42bdd](https://github.com/Supernova-Studio/exporter-documentation/pull/166/commits/da42bddecd96804f218f184aa0ce3982fa1b75fc)